data/json/items/fluff.json

```diff
@@ -31,20 +31,35 @@
     "flags": [ "NO_REPAIR" ],
     "use_action": [ "PLAY_GAME" ],
     "variants": [
+      {
+        "id": "generic_deck_of_cards",
+        "name": { "str": "deck of cards", "str_pl": "decks of cards" },
+        "description": "A fairly new, glossy deck of 52 playing cards.",
+        "weight": 30
+      },
+      {
+        "id": "deck_of_cards_worn",
+        "name": { "str": "worn deck of cards", "str_pl": "worn decks of cards" },
+        "description": "A deck of 52 playing cards.  Judging by the bent, frayed edges and faded prints, it saw its share of games.",
+        "weight": 6
+      },
```

Comment:
Although I get the intent here, I think "worn" decks of cards should be represented as `deck of cards |\` instead of `worn deck of cards ||`.

```suggestion
```

Reply:
Counterpoint: there's a mangled teddy bear variant, and plenty of other things like that. It's just flavor, I think representing it as durability makes sense when it's an item where durability matters, like armor or a weapon.

Reply:
1. Looks like a good argument to remove the mangled teddy bear then. 
2. I would argue that the durability **is** the fluff, there is no fluff difference between a `deck of cards |.` and `worn deck of cards ||`, you just want that fluff to be in the name of the item. 
3. I would also argue that there is no item where "durability doesn't" matter, durability is a feature of all items because all items can be damaged, and all items can be damaged because damage is essential to fluff and gameplay.
4. Finally, a "worn" item should be closer to it's next state, "unusable" due to damage, those are the gameplay consequences mentioned in point 3, although this is a weak argument because in theory this issue could be solved my making the "worn"version have less durability total.

In the end, it's your item, I'm not a reviewer, this is an extreme nitpick and I don't wish to antagonize you, feel free to resolve the conversation.